src/screens/Profiles/UserLibrary/components/UserLibraryEditScreen/component.js

```diff
@@ -151,128 +175,139 @@ export class UserLibraryEditScreenComponent extends React.Component {
           rightAction={this.saveEntry}
         />
 
-        <View style={styles.editRow}>
-          <View>
-            <Text style={[styles.editRowLabel, styles.withValueLabel]}>
-              Library Status
-            </Text>
-            <Text style={styles.editRowValue}>
-              {status}
+        <KeyboardAwareScrollView>
+          {/* Status */}
+          <View style={styles.editRow}>
+            <Text style={styles.editRowLabel}>
+              Library Status:
             </Text>
-          </View>
-          {canEdit &&
-            <SelectMenu options={this.selectOptions} onOptionSelected={this.onStatusChanged}>
-              <Text>X</Text>
+            <SelectMenu
+              disabled={!canEdit}
+              options={this.selectOptions}
+              onOptionSelected={this.onStatusChanged}
+            >
+              <Text style={styles.editRowValue}>
+                {status}
+              </Text>
             </SelectMenu>
-          }
-        </View>
-        <View style={styles.editRow}>
-          <Text style={styles.editRowLabel}>
-            {libraryType === 'anime' ? 'Episode' : 'Chapter'} Progress:
-          </Text>
-          <Counter
-            disabled={!canEdit}
-            initialValue={libraryEntry.progress}
-            maxValue={typeof maxProgress === 'number' ? maxProgress : undefined}
-            progressCounter={typeof maxProgress === 'number'}
-            onValueChanged={this.onProgressValueChanged}
-          />
-        </View>
-        <View style={styles.editRow}>
-          <Text style={styles.editRowLabel}>Rating:</Text>
-          <Rating
-            disabled={!canEdit}
-            size="large"
-            viewType="single"
-            onRatingChanged={this.onRatingChanged}
-            style={styles.ratingStyle}
-            ratingTwenty={this.state.ratingTwenty}
-            ratingSystem={ratingSystem}
-          />
-        </View>
-        <View style={styles.editRow}>
-          <Text style={styles.editRowLabel}>Times {libraryType === 'anime' ? 'Rewatched' : 'Read'}:</Text>
-          <Counter
-            disabled={!canEdit}
-            initialValue={libraryEntry.reconsumeCount}
-            onValueChanged={this.onTimesRewatchedChanged}
-          />
-        </View>
-        <View style={styles.editRow}>
-          <View>
-            <Text style={[styles.editRowLabel, styles.withValueLabel, { paddingBottom: 4 }]}>
-              Library Entry Visibility
+          </View>
+          {/* Progress */}
+          <View style={styles.editRow}>
+            <Text style={styles.editRowLabel}>
+              {libraryType === 'anime' ? 'Episode' : 'Chapter'} Progress:
             </Text>
-            <Text style={styles.editRowValue}>
-              {this.state.private ? 'Private' : 'Public'}
+            <Counter
+              inputRef={(r) => { this.progressInput = r; }}
```

Comment:
would adding [blurOnSubmit](https://facebook.github.io/react-native/docs/textinput.html#bluronsubmit) to `Counter`'s `TextView` help?